### DISCUSSION TITLE
If I need to integrate external websocket communication LSP service in vscode, can I implement this plug-in? Thank you so much

### DISCUSSION THREAD

**Discussion by sailthesea** [microsoft/vscode-discussions#1271] [Extension Development QnA]
For example: ws://192.168.210.87:3000?languageId=python, which provides a websocket address through which you can directly access the LSP service.

**Comment by isidorn** [comment]
Can you re-phrase your question?

**Comment by olguzzar** [comment]
Closing this discussion as we need more information to assist - please feel free to reopen if you have new details!
